baseapp-chats/baseapp_chats/graphql/mutations.py

```diff
@@ -168,6 +168,16 @@ def mutate_and_get_payload(cls, root, info, profile_id, participants, is_group,
         room.participants_count = len(created_participants)
         room.save()
 
+        if is_group:
+            send_message(
+                message_type=Message.MessageType.SYSTEM_GENERATED,
+                room=room,
+                profile=None,
+                user=None,
+                content='{content_linked_profile} created group "' + title + '"',
```

Comment:
lets put that on a constant, we'll probably need when sending new system messages